sysutils/ctop/Portfile

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        bcicen ctop 0.7.5 v
+github.tarball_from releases
+revision            0
+
+categories          sysutils devel
+platforms           darwin
+supported_archs     x86_64
+license             MIT
+maintainers         {outlook.de:judaew @judaew} openmaintainer
+
+description         Top-like interface for container metrics
+long_description    \
+    ctop provides a concise and condensed overview of real-time metrics for \
+    multiple containers.\nctop comes with built-in support for Docker and \
+    runC\; connectors for other container and cluster systems are planned for \
+    future releases.
+
+distname            ctop-${version}-darwin-amd64
+extract.suffix
+
+extract {
+    exec mkdir ${worksrcpath}
+    file copy ${distpath}/${distfiles} ${worksrcpath}/${name}
+}
+
+checksums           rmd160  f81e9814dcf096d4293a27bc42fca78a829a21a1 \
+                    sha256  bb7e51429ff03a2c065873c3bd9e7448af1b9733a4b58b71de12bc4a728879de \
+                    size    11529016
+
+use_configure       no
+
+build {}
```

Comment:
`ctop` should be buildable from source using the `golang` portgroup.

```
build.cmd      make
build.pre_args VERSION=${version}
build.args     build
```

You can choose to vendor dependencies using the `go2port` tool, but if that does not work, you can remove `GO111MODULE=off` and `GOPROXY=off` from `build.env`.

Reply:
Building it from source also means that you can support any arch that MacPorts' Go will build on, allowing you to remove `supported_archs x86_64`.